### PR TITLE
Improve offense messages

### PR DESCRIPTION
**Pull request by utkarsh2102**
Make it a bit more clearer as to what's
problematic and what to do intead of those
lines of code.

Signed-off-by: Utkarsh Gupta <<utkarsh@debian.org>>

### CODE REVIEW

File: config/default.yml [hunk 1]
```diff
@@ -1,25 +1,36 @@
...
+                 Using `bundler/setup` in tests is redundant. Consider
+                 removing it.
   Enabled: true
   VersionAdded: '0.4'
   VersionChanged: '0.5'
 
 Packaging/GemspecGit:
-  Description: 'Use pure Ruby alternative instead of `git ls-files`.'
+  Description: >-
+                 Using `bundler/setup` in tests is redundant. Consider
```
Comment (deivid-rodriguez) [inline config/default.yml:13]: You probably didn't mean to mention `bundler/setup` here.

Comment (utkarsh2102) [reply config/default.yml:13]: Aw, crap! I forgot this here 🤦🏻 
Would you have time to just make a straight-forward commit fixing this?

Comment (utkarsh2102) [reply config/default.yml:13]: Just btw, this is a pretty impressive catch, thanks! 😄

Comment (deivid-rodriguez) [reply config/default.yml:13]: Sorry, I'm a bit buried right now, I'm just quickly going through notifications and saw this, so just wanted to let you now.

Comment (utkarsh2102) [reply config/default.yml:13]: No problem, fixed via a9925e7f7e6624e961965876684f5c0ebb05f487.

**Pull request merged**